CONTRIBUTING.md

```diff
@@ -67,19 +67,19 @@ It is important that "origin" points to your fork.
 
 * Make sure your are starting with the branch for the latest version, i.e., _grass7_:
```

Comment:
Since we are already in grass 8, perhaps it's better to base the examples on grass8 branch

Reply:
Solved in the current version.